manifests/tar_extract.pp

```diff
@@ -8,10 +8,9 @@
 # Actions:
 # - Extracts a tarball
 #
-# Requires:
-# - The certs class
-#
 define certs::tar_extract($path = $title) {
+  validate_file_exists($path)
```

Comment:
Is there a way we can add a message to say, the file does not exist here, and let them know how to regen a certs tar?

Reply:
+1 was just curious, I don't consider this a blocker.